aiida-registry-app/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "aiida-registry",
+  "private": true,
+  "version": "0.0.0",
```

Comment:
This is the react component version correct? What is your plan to set it? Is this version used or we can just remove it?

Reply:
Not sure how to use it so will remove it. I think this is needed if users will use a specific version via npm, not applicable for this project, right?